lib/plugin-compat.php

```diff
@@ -25,10 +25,12 @@
  * @param  array $post      Post object which contains content to check for block.
  * @return array $post      Post object.
  */
-function gutenberg_remove_wpcom_markdown_support( $post ) {
-	if ( gutenberg_content_has_blocks( $post->post_content ) ) {
-		remove_post_type_support( 'post', 'wpcom-markdown' );
+function gutenberg_remove_wpcom_markdown_support( $post_data ) {
+	if ( gutenberg_content_has_blocks( implode( "\n", $post_data ) ) ) {
+        if ( class_exists ( 'WPCom_Markdown' ) ) {
```

Comment:
No need for nesting `if`, just combine into the top-level condition.

Reply:
Fixed.